.github/workflows/build.yaml

```
@@ -252,6 +252,8 @@ jobs:
             image/rhel/docs
 
   check-collector-images-exist:
+    # This job ensures that COLLECTOR_VERSION cannot be updated to a version for which
+    # the image was not successfully built on GHA and Konflux.
```

If we're to really check GHA, we should include `quay.io/stackrox-io/` repos. Like [I mentioned earlier](https://github.com/stackrox/stackrox/pull/13502#discussion_r1876090857), we did not need this at all when GHA was the only upstream build CI. I suggest removing the claim about GHA (here and for scanner job).

```suggestion
    # the image was not successfully built on Konflux.
```